addons/web/static/src/views/fields/radio/radio_selection_field_with_filter.js

```diff
@@ -7,11 +7,11 @@ import { registry } from "@web/core/registry";
 export class RadioSelectionFieldWithFilter extends RadioField {
     static props = {
         ...RadioField.props,
-        allowed_selection: { type: Array },
+        allowed_selection: { type: Set },
```

Comment:
Normally we shouldn't change this in stable, but I guess 17.2 is young enough ^^